azkaban-common/src/main/java/azkaban/user/UserUtils.java

```diff
@@ -1,7 +1,25 @@
 package azkaban.user;
 
+import com.google.common.base.Preconditions;
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.Multimap;
+import com.sun.nio.file.SensitivityWatchEventModifier;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardWatchEventKinds;
+import java.nio.file.WatchEvent;
+import java.nio.file.WatchKey;
+import java.nio.file.WatchService;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.log4j.Logger;
```

Comment:
sl4fj is preferred. see: https://github.com/azkaban/azkaban/blob/master/CONTRIBUTING.md